docs/createFlows.rst

```diff
@@ -0,0 +1,134 @@
+.. _CreatingFlows:
```

Comment:
When do you plan to release conditional workflow docs?

Reply:
It's not yet ready. Still need to add some changes to it, so will release it later.